python-package/xgboost/dask.py

```diff
@@ -639,14 +639,15 @@ async def _train_async(client,
 
     workers = list(_get_workers_from_data(dtrain, evals))
     _rabit_args = await _get_rabit_args(len(workers), client)
+    _global_config = config.get_config()
```

Comment:
Is it okay that `get_config()` is not async, whereas `_get_rabit_args()` is?

Reply:
Get rabit args has a client running in it.